src/instructlab/model/backends/backends.py

```diff
@@ -39,6 +40,13 @@
 API_ROOT_WELCOME_MESSAGE = "Hello from InstructLab! Visit us at https://instructlab.ai"
 CHAT_TEMPLATE_AUTO = "auto"
 CHAT_TEMPLATE_TOKENIZER = "tokenizer"
+
+# Internal env variable for CI adjustment / disablement
+# pylint: disable=unnecessary-direct-lambda-call
+MAX_STABLE_VRAM_WAIT = (lambda value: int(value) if value and value.isdigit() else 30)(
```

Comment:
```
>>> "𐩃".isdigit()
True
>>> int("𐩃")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '𐩃'
```

I think it's safer to just:

```
try:
    MAX_STABLE_VRAM_WAIT = int(value)
except ValueError: # perhaps some more types?
    MAX_STABLE_VRAM_WAIT = 30
```

And yes, I realize that this is a very weird case that probably no one would even consider.

Reply:
Or use `isdecimal` maybe.

Reply:
@booxter great catch i was thinking decimal and for some reason typed digit 🤦 .it turns out to make this easier to test I need to just make this a function